components/builder-core/src/metrics.rs

```diff
@@ -50,17 +49,6 @@ pub trait CounterMetric: Metric {
     }
 }
 
-pub trait GaugeMetric: Metric {
```

Comment:
Should GaugeMetric be restored?

Reply:
probably not if we don't use it anymore.

Reply:
It did seem to be used anywhere and removal hasn't caused any issues I'm aware of but I was worried that I might be missing something about the stats.  Consider it gone.